src/GNDStk/Node/test/meta.test.cpp

```diff
@@ -0,0 +1,48 @@
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+using namespace njoy::GNDStk;
+
+SCENARIO("Testing GNDStk Node meta()") {
+
+   GIVEN("The top-level node from a tree object") {
+      Tree<> tree("n-069_Tm_170-covar.xml");
```

Comment:
This form of populating data works. However, in the future, I'd rather just have some hard-coded string in the test file. That way we don't have to keep MB of data around when we need just a few kB. 

Of course, if you are testing reading from a file, then we have to have a separate file.

Reply:
It looks like there are three different .xml files in one place or another in our test suite: n-008_O_016.xml, n-069_Tm_170-covar.xml, and n-026_Fe_056.xml. I think we talked about this at some point, and decided that having those three was appropriate. If git compresses things, then, well, the good news is that text-based files (like XML) compress pretty well. Of note, n-008 appears twice (in Node/ resources and Tree/ resources), and n-069 appears three times (in Node/ resources, Tree/ resources, and keyword/ resources). Perhaps a redesign of the build and test systems would allow us to have each .xml only once, in some common area. Note also that n-026, which appears only once, is (at about 17 megs) by far the largest of the three .xml files in our test system. A case could be made that having one relatively large test file is a good thing, as it shows that we can process "real", substantial GNDS input.